### PR TITLE
Update db.json

### PR DESCRIPTION
**Pull request by it-isa-me-mario**
## Changes proposed

Some abbreviations didn't have a closing curly bracket "}"

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: public/server/db.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
     },
```
Comment (Njong392) [inline public/server/db.json:97]: Oops. I think there is a trailing bracket here.
```suggestion
    
```

**Review by Njong392 — CHANGES_REQUESTED** [review]
What a nice spot! Thank you for looking over this. Do check out the suggested changes I have made.

File: public/server/db.json (hunk 1 above)
Comment (it-isa-me-mario) [reply public/server/db.json:97]: Thank you for your encouragement! 

Looking at "Byob" that starts from line 94 and ends at line 97, it looks all good?

Comment (Njong392) [reply public/server/db.json:97]: > Thank you for your encouragement!
> 
> Looking at "Byob" that starts from line 94 and ends at line 97, it looks all good?

Ah yes. Looks like a miscalculation on my part. You have a good eye haha. Thanks again for the contribution! I'll go ahead and merge this.

**Pull request merged**